reaktive/src/jsMain/kotlin/com/badoo/reaktive/utils/SynchronizedCompat.kt

```diff
@@ -0,0 +1,3 @@
+package com.badoo.reaktive.utils
+
+internal actual inline fun <T> synchronizedCompat(lock: Any, block: () -> T): T = block()
```

Comment:
Should not `synchronized` be no-op in JS by default?
Or it was removed?

Reply:
It was deprecated for a long time, and now doesn't compile in a shared source set. However it's still present in JS, and deprecated. Not sure what is the actual reason, but better to not use deprecated things anyway.